packages/web/package.json

```diff
@@ -23,6 +23,7 @@
     "preview:prod": "npm run build:prod && env-cmd ./env/.env.prod vite preview --outDir build-production",
     "preview:stage": "npm run build:stage && env-cmd ./env/.env.stage vite preview --outDir build-staging",
     "preview:stage:ci": "env-cmd ./env/.env.stage vite preview --outDir build-staging",
+    "preview:dev": "npm run build:dev && env-cmd ./env/.env.dev vite preview --outDir build-development",
```

Comment:
so this is the "build version" of dev so we dont have to have the vite requests for every .ts file?

Reply:
Yup exactly! we were already doing this for stage